docs/references/protocol/ledger-data/ledger-entry-types/did.md

```diff
@@ -12,7 +12,7 @@ status: not_enabled
 
 _(Requires the [DID amendment][] {% not-enabled /%})_
 
-A `DID` ledger entry holds references to, or data associated with, a single [DID](../../../../concepts/accounts/decentralized-identifiers.md).
+A `DID` ledger entry holds references to, or data associated with, a single [DID](/docs/concepts/decentralized-storage/decentralized-identifiers.md).
```

Comment:
If absolute paths are used, the link is forced to be to an English page, resulting in poor usability on untranslated pages.